test/views/pages/profile/profile_page_test.dart

```diff
@@ -191,6 +200,26 @@ void main() {
       final centauriText = find.text("username_8456 · 0 Centauri");
       expect(centauriText, findsOneWidget);
     });
+
+    testWidgets("Posts refreshing works correctly", (tester) async {
+      await tester.pumpWidget(mockedProfilePage);
+      await tester.pumpAndSettle();
+
+      // Check that the profile page is displayed
+      final profilePage = find.byType(ProfilePage);
+      expect(profilePage, findsOneWidget);
+
+      // Check that the post info column is displayed
+      final postColumn = find.byKey(ProfileUserPosts.postColumnKey);
+      expect(postColumn, findsOneWidget);
+
+      // Refresh the user posts
+      await tester.fling(postColumn, const Offset(100, 400.0), 1000.0);
+      await tester.pumpAndSettle(delayNeededForAsyncFunctionExecution);
+
+      // Check that refreshing washandled correctly
+      expect(find.byKey(ProfileUserPosts.postColumnKey), findsOneWidget);
+    });
```

Comment:
Here, would not be a good idea to check that the mock post that we are creating in the set-up is actually showing before and after the refresh.

```     
// Check that the post card is displayed
final postCard = find.byKey(ProfileInfoCard.infoCardKey);
expect(postCard, findsOneWidget);

Reply:
Yes, added in a49da69